CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
 
-Talking to us first about the enhancement you want to build will be the most likely way to get your pull request into the library (see Our Contributing Philosophy above). We would hate to see you write code you’re proud of, just to learn that we’ve already been working on the same thing, or that we feel doesn’t fit into the core library.
+Talking to us first via the discussions section about the enhancement you want to build will be the most likely way to get your pull request into the library (see Our Contributing Philosophy above). We would hate to see you write code you’re proud of, just to learn that we’ve already been working on the same thing, or that we feel doesn’t fit into the core library.
 
-### Contact us
+Once your idea has been selected for the next release of LabVIEW, a branch will be created that you can submit your pull request to.
 
-To ask about a possible enhancement, email us at {team_email_address}
+## Creating a discussion for a new feature
 
-### 💫 Pull Requests
+The first step to start an enhancement to the icon editor is to create a discussion on the [New Features](https://github.com/ni/labview-icon-editor/discussions/categories/new-features) discussion board. You must describe the problem that made you want to have this enhancement.
 
-If you want to contribute improvements or new features, we are happy to review your PR :)  
-Please use a meaningful commit message and add a little description of your changes.
-
-### Configuring your Development Environment
+### 💫 Pull Requests
 
-1. Install package dependencies `dragon install`
-2. {step_2}
-3. Test your changes by editing the icons of some VIs
-4. Run tests `caraya` and add new tests if you are introducing a new feature
+When submitting a PR, please follow these guidelines to ensure clarity and ease
+of review:
+ 
+1. **Title**: Start with a short, descriptive title that summarizes the change.
+2. **Description**:
+   - **Purpose**: Explain why you are making this change.
+   - **Changes Made**: Describe what changes you made and why.
+   - **Related Issues**: Reference any related issues using `#issue_number`.
+   - **Testing**: Outline how you tested your changes and any specific areas of
+     the code to focus on during review.
+   
+Please remember to sign off your commits (e.g., by using git commit -s if you
+are using the command line client). This amends your git commit message with a
+line of the form Signed-off-by: Name Lastname <name.lastmail@emailaddress.com>.
+Please include all authors of any given commit into the commit message with a
+Signed-off-by line. This indicates that you have read and signed the Developer
+Certificate of Origin (see below) and are able to legally submit your code to
+this repository.
+
+### Reporting Issues
```

Comment:
@svelderrainruiz I'm not at my computer, but if you remind me, I can point you to how to set up templates for issues/pull-requests